notebooks_tsqr/consdb/assorted_plots.yaml

```diff
@@ -0,0 +1,18 @@
+# For use with a Times Square notebook
+title: Assorted Plots
+description: Trying plots
+authors:
+  - name: Valerie Becker
+    slack: valerie becker
+tags:
+  - reporting
+  - prototype
+parameters:
+  day_obs:
+    type: string
+    description: Date on which to query
+    default: '2024-06-26'
```

Comment:
Probably should change the default.  utils.get_datetime_from_day_obs_str will let you use TODAY, TOMORROW, YESTERDAY also.

Reply:
That's a good point, however because of the kernel issues I cannot yet use anything in our own lsst package yet while being in the lsst kernel, as my pages for consdb are.